src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -377,7 +377,7 @@ public boolean canParse(final ParseContext context) {
 					
 					@Override
 					public String toString(final Location l, final int flags) {
-						return "x: " + Skript.toString(l.getX()) + ", y: " + Skript.toString(l.getY()) + ", z: " + Skript.toString(l.getZ());
+						return "x: " + Skript.toString(l.getX()) + ", y: " + Skript.toString(l.getY()) + ", z: " + Skript.toString(l.getZ()) + ", yaw: " + Skript.toString(l.getYaw()) + ", pitch: " + Skript.toString(l.getPitch()) + " in '" + l.getWorld().getName() + "'";
```

Comment:
I think it would be good to check whether the world is null, and if it is, to not include it 🙂

Reply:
Sure \:D